lib/window.ts

```diff
@@ -35,8 +35,8 @@ module.exports = function createWindow(currentWindow, channel) {
     currentWindow.removeEventListener('resize', autoUpdateHeight)
   }
 
-  function updateHeight(height) {
-    if (height == null) {
+  function updateHeight(height: number = null) {
```

Comment:
```suggestion
  function updateHeight(height: number|null = null) {
```
I think now we're passing because of the permissive setup, but it won't pass later on